### PR TITLE
Typoes fix.

### PR DESCRIPTION
**Pull request by aviweit**
Hi,

I opened this PR to fix several typos in the documentation that I have encountered during Paraglider installation process.

Thanks.

### CODE REVIEW

File: docs/source/examples/controller-setup.rst [hunk 1]
```diff
@@ -19,11 +19,11 @@ The controller is configured using a configuration file. The configuration file
...
         - name: "gcp"
-            host: "localhost"
-            port: 8082
+          host: "localhost"
+          port: 8082
         - name: "azure"
-            host: "localhost"
-            port: 8083
+          host: "localhost"
+          port: 8083
```
Comment (praveingk) [inline docs/source/examples/controller-setup.rst:27]:
```suggestion
        - name: "ibm"
          host: "localhost"
          port: 8084
```

File: docs/source/examples/controller-setup.rst [hunk 2]
```diff
@@ -32,7 +32,7 @@ The controller is configured using a configuration file. The configuration file
             - name: "gcp"
               deployment: "projects/${GCP_PROJECT_ID}"
             - name: "ibm"
-              deployment: "resource_groups/${IBM_RESOURCE_GROUP}"
+              deployment: "/resourcegroup/${IBM_RESOURCE_GROUP}"
 
     tagService:
         host: "localhost"
```
Comment (praveingk) [inline docs/source/examples/controller-setup.rst:38]: Please change the port of tag service and kvstore to 8085 and 8086 respectively.

**Review by praveingk — DISMISSED** [review]
Thanks @aviweit , Looks good, I just had a suggestion to add ibm plugin host/port was missing in the declaration, and change the ports of other service accordingly.

**Comment by aviweit** [comment]
Hi @praveingk , I accepted your change suggestion and committed port ordering.
Thanks.

**Review by seankimkdy — APPROVED** [review]
Hey @aviweit thanks for catching these! Your build is failing because of your GitHub token permissions as an external contributor (i.e., PR from a fork), but it's an issue on our end not yours. We're trying to figure out what the best solution is, so hold tight.

**Review by praveingk — APPROVED** [review]

**Comment by seankimkdy** [comment]
@aviweit While we work on getting the right permissions for running the builds, can you amend your commits to pass our DCO workflow? https://github.com/paraglider-project/paraglider/pull/332/checks?check_run_id=25242666216 has instructions.

For the future, I recommend [setting up the git hook](https://paragliderproject.io/developers/contributing.html#developer-certificate-of-origin).

**Comment by aviweit** [comment]
@seankimkdy , thank you for the comments. I have amended the commits to include signed-off.

**Pull request merged**